### DISCUSSION TITLE
issue

### DISCUSSION THREAD

**Discussion by 1wtbb** [projectdiscovery/katana#339] [Q&A]
![image](https://user-images.githubusercontent.com/87883732/222896886-d080ecad-9ecc-4a4d-9986-282ca34d778b.png)
I don't know why its output is empty

**Comment by ehsandeep** [comment]
@NezHacker please follow GH reporting template to include the required information to file a bug report.

for this issue, please include the output of `katana -hc` in the comment.

**Comment by 1wtbb** [comment]
![image](https://user-images.githubusercontent.com/87883732/222897377-c9340572-8f7e-48f4-8c8c-7aa40eef20ce.png)

**Comment by 1wtbb** [comment]
It seems that my vps can't access scanme. sh: 80

**Comment by ehsandeep** [comment]
@NezHacker, it seems your vps doesn't have access to the external network, so none of the tools working as well, including - https://github.com/projectdiscovery/pdtm/issues/85

**Reply by 1wtbb** [reply]
![image](https://user-images.githubusercontent.com/87883732/222898075-d7c64a76-cd5f-4734-b7e5-c1836013a92d.png)

**Comment by 1wtbb** [comment]
My vps can communicate with the internet, but the terminal still doesn't output anything. Why

**Comment by 1wtbb** [comment]
I can ping scanme.sh, and I'm sure my vps can be connected.. Because I also use many other tools. However, the display here cannot be connected. I don't know why

**Comment by MetzinAround** [comment]
answered in https://github.com/projectdiscovery/pdtm/issues/85
